events/wc_frostwolf_events.txt

```diff
@@ -1086,34 +1127,289 @@ narrative_event = { #Frostwolf settled
 	}
 	option = {
 		trigger = {
-			NOT = { character = ROOT }
+			NOT = { character = event_target:target_frostwolf_leader }
 			culture_group = orc_group
 		}
-		name = EVTOPTC_WCFW.45 #These rats cannot hide from the Horde!
+		name = EVTOPTC_WCFW_50 # Rats! The Horde will hunt them all!
 		piety = 5
 	}
 }
 
-# FAIL EVENT
-character_event = {
-	id = WCFW.200
-	picture = GFX_evt_orcish_horde
-	desc =  EVTDESC_WCFW_200
-	border = GFX_event_normal_frame_war
+# Frostwolf failure event 60-63
+# delayer event
+province_event = {
+	id = WCFW.160
 	
 	is_triggered_only = yes
+	hide_window = yes
 	
 	immediate = {
 		d_frostwolf = {
-			clr_title_flag = fw_hunger_event_flag
+			holder_scope = {
+				if = {
+					limit = { frostwolf_leader_exists = yes }
+					narrative_event = { id = WCFW.60 }
+				}
+			}
 		}
 	}
+}
+
+# Frostwolf failure popup (will always happen, even if Stormreaver is dead)
+# delayed event
+narrative_event = {
+	id = WCFW.60
+	title = EVTTITLE_WCFW_60
+	desc =  EVTDESC_WCFW_60 # It seems that despite all the efforts your clan has made to leave the Horde, they have all failed.
+	picture = GFX_evt_orc_city
+	border = GFX_event_narrative_frame_war
+
+	is_triggered_only = yes
 
 	option = {
-		name = I_SEE
+		name = EVTOPT_WCFW_60 # We are Frostwolves, we will endure.
+		73 = {
+			province_event = {
+				id = WCFW.161
+				days = 7
+			}
+		}
+	}
+}
+
+# delayer event
+province_event = {
+	id = WCFW.161
+	
+	is_triggered_only = yes
+	hide_window = yes
+	
+	trigger = {
+		d_frostwolf = {
+			holder_scope = {
+				frostwolf_leader_exists = yes
+			}
+		}
+	}
+	
+	immediate = {
+		d_stormreaver = {
+			holder_scope = {
+				if = {
+					limit = { frostwolf_chain_butcher_exists = yes }
+					narrative_event = { id = WCFW.61 }
+				}
+			}
+		}
 	}
 }
 
+# Forcer informed of failure event (this can fail if Stormreaver is dead)
+# delayed event
+narrative_event = {
+	id = WCFW.61
+	title = EVTTITLE_WCFW_61
+	desc =  EVTDESC_WCFW_61 # Despite the fairly clear orders you gave to [From.GetFirstName], the Frostwolf clan refused to leave the Horde.
+	picture = GFX_evt_orc_city
+	border = GFX_event_narrative_frame_war
 
+	is_triggered_only = yes
 
+	option = {
+		name = EVTOPT_WCFW_61 # What is necessary shall be done.
+		73 = {
+			province_event = {
+				id = WCFW.162
+				days = 7
+			}
+		}
+	}
+}
 
+# delayer event
+province_event = {
+	id = WCFW.162
+	
+	is_triggered_only = yes
+	hide_window = yes
+	
+	trigger = {
+		frostwolf_leader_exists = yes
```

Comment:
Wrong place for this trigger. `frostwolf_leader_exists` checks character conditions, not provinces.

Reply:
This should be resolved.